cosmwasm/contracts/crosschain-registry/src/state.rs

```diff
@@ -47,6 +49,19 @@ impl<T: AsRef<str>> From<(T, bool)> for RegistryValue {
     }
 }
 
+#[cw_serde]
+#[derive(Default)]
+pub struct ChainPFM {
+    pub acknowledged: bool,
+    pub validated: bool,
```

Comment:
What is the difference between the two booleans? Consider adding comments

Reply:
acknowledged = the IBC ack has been received.
validated = the validate_pfm wasm hook has been called (to check that what we received is what we expected) and the data has been updated.